src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -447,21 +493,9 @@ private static bool IsAfterDot(ITextSnapshot snapshot, ITrackingSpan applicableT
             // See which filters might be enabled based on the typed code
             var textFilteredFilters = filteredList.SelectMany(n => n.VSCompletionItem.Filters).ToImmutableHashSet();
 
-            // When no items are available for a given filter, it becomes unavailable
-            return ImmutableArray.CreateRange(filters.Select(n => n.WithAvailability(textFilteredFilters.Contains(n.Filter))));
-        }
-
-        private static bool ShouldBeFilteredOutOfCompletionList(VSCompletionItem item, ImmutableArray<CompletionFilter> activeFilters)
-        {
-            foreach (var itemFilter in item.Filters)
-            {
-                if (activeFilters.Contains(itemFilter))
-                {
-                    return false;
-                }
-            }
-
-            return true;
+            // When no items are available for a given filter, it becomes unavailable.
+            // Expanders always appear available as long as it's presented.
+            return ImmutableArray.CreateRange(filters.Select(n => n.WithAvailability(n.Filter is CompletionExpander ? true : textFilteredFilters.Contains(n.Filter))));
```

Comment:
>filters.Select [](start = 46, length = 14)

filters.SelectAsArray. Then, you do not need mmutableArray.CreateRange

Reply:
Fixed